### DISCUSSION TITLE
Application décomissionnée

### DISCUSSION THREAD

**Discussion by adrienloyat** [dbarzin/mercator#991] [Q&A] [answered]
Bonjour
Existe-t-il un moyen d'indiqué qu'une application est en cours de décomissionnement, voir plus utilisée, mais sans avoir à supprimer l'application ? 
Merci d'avance

**Comment by dbarzin** [comment, marked as answer]
Vous pouvez ajouter un attribut à l'application. Par exemple "DECOM" pour signaler une application décommissionnée.

**Reply by adrienloyat** [reply]
Ce semble bien. Merci
